README.md

```diff
@@ -6,22 +6,45 @@
 
 ## Installation
 
+### Prerequisities
+
+* GNU-Make
+* Python && pip
+* OpenJDK (required by submodule)
+* optional: Docker CE for building container images 
+
 Clone the repository:
 
 ```sh
-git clone ocrd_fileformat
-cd ocrd_fileformat
+git clone --recursive git@github.com:OCR-D/ocrd_fileformat.git
 ```
 
-Update submodules:
+Step into local clone, build and install `ocr-fileformat` and the `ocrd_fileformat` wrapper:
 
 ```sh
-git submodule sync
-git submodule update --init
+cd ocrd_fileformat
+make install-fileformat install
 ```
 
-Build and install `ocr-fileformat` and the `ocrd_fileformat` wrapper:
+## Usage
 
-```sh
-make install-fileformat install
+After successful installation type `ocrd-fileformat --help` to get an idea which conversions can be achived.
+
+With the OCR-D-CLI-Wrapper OCR-format conversions integrate fluently into existing OCR-D-Tool-Workflows.
+
+Given a previous step `OCR`, which produces default PAGE-XML, a conversion from PAGE-XML into plain text can be achieved with:
+
+```Makefile
```

Comment:
This assumes users are using https://github.com/bertsky/workflow-configuration. The generic CLI would be best here, with instructions for workflow-configuration/ocrd_framework/my_ocrd_workflow maybe in details/summary elements?


```html
<details>
  <summary>ocrd-fileformat-transform -I OCR -O OCR-TXT -p '{"from-to": "page-text"}'</summary>
  <details>
      <summary>workflow-configuration instructions</summary>
      <pre>
CONVERT = OCR-TXT
$(CONVERT): $(OCR)
$(CONVERT): TOOL = ocrd-fileformat-transform
$(CONVERT): PARAMS = "from-to" : "page text"
      </pre>
  <details>
</details>
 
</details>
```

<details>
  <summary><code>ocrd-fileformat-transform -I OCR -O OCR-TXT -p '{"from-to": "page-text"}'</code></summary>
  <details>
      <summary>workflow-configuration instructions</summary>
      <pre>
CONVERT = OCR-TXT
$(CONVERT): $(OCR)
$(CONVERT): TOOL = ocrd-fileformat-transform
$(CONVERT): PARAMS = "from-to" : "page text"
      </pre>
  <details>
</details>
 
</details>

We should also use some such convention for the user documentation IMHO

Reply:
I enhanced the usage section, but I don't get what I'm supposed to do with the XML-snippet above?

Reply:
Yeah, that is not really important here, snippet was more for discussion how we can get all current workflow solutions covered in the docs on the website.